modules/core/src/api/profile.ts

```diff
@@ -115,7 +113,6 @@ export const getProfileInfo = (endpoint: string,
             const profileResponse: ProfileInfoInterface = {
                 emails: response.data.emails || "",
                 name: response.data.name || { familyName: "", givenName: "" },
-                organisation: response.data[orgKey] ? response.data[orgKey].organization : "",
```

Comment:
why this line was removed?

Reply:
```getProfileInfo``` has no active use , since ```orgKey```  variable is removed line 118 ```organisation: response.data[orgKey] ? response.data[orgKey].organization : "",``` also removed